core/config/toml/types.go

```diff
@@ -95,18 +96,25 @@ func (c *Core) SetFrom(f *Core) {
 	c.Sentry.setFrom(&f.Sentry)
 	c.Insecure.setFrom(&f.Insecure)
 	c.Tracing.setFrom(&f.Tracing)
+	c.Telemetry.setFrom(&f.Telemetry)
 }
 
 func (c *Core) ValidateConfig() (err error) {
 	_, verr := parse.HomeDir(*c.RootDir)
-	if err != nil {
+	if verr != nil {
 		err = multierr.Append(err, configutils.ErrInvalid{Name: "RootDir", Value: true, Msg: fmt.Sprintf("Failed to expand RootDir. Please use an explicit path: %s", verr)})
 	}
 
 	if (*c.OCR.Enabled || *c.OCR2.Enabled) && !*c.P2P.V2.Enabled {
 		err = multierr.Append(err, configutils.ErrInvalid{Name: "P2P.V2.Enabled", Value: false, Msg: "P2P required for OCR or OCR2. Please enable P2P or disable OCR/OCR2."})
 	}
 
+	if *c.Tracing.Enabled && *c.Telemetry.Enabled {
```

Comment:
If both are enabled will there be a clash over how the globally registered tracing provider is set?

Reply:
No, `TraceSpanExporter` is used to register a second exporter.

I did notice an issue with the attributes merge though. Fix incoming.

Reply:
Common fix here: https://github.com/smartcontractkit/chainlink-common/pull/763